config/routes.rb

```diff
@@ -1,11 +1,19 @@
 Rails.application.routes.draw do
   namespace :api do
     namespace :v1 do
+      # All events
       resources :events, only: [:index]
+
+      # All users
       resources :users, only: [:create]
+
+      # Specific User
       namespace :user do
         resources :events, only: [:index]
       end
+
+      # User Login
+      post '/sessions', to: 'sessions#create'
```

Comment:
Logan can you run me through what is happening on line 16,  curious about the post here.